org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/contentmodel/participants/codeactions/NoGrammarConstraintsCodeAction.java

```diff
@@ -101,17 +103,26 @@ public void doCodeAction(Diagnostic diagnostic, Range range, DOMDocument documen
 					diagnostic);
 			codeActions.add(dtdWithXmlModelAction);
 
+			// ---------- Open Binding Wizard
+			if (sharedSettings.getBindingWizardSupport()) {
+				String documentURI = document.getDocumentURI();
+				String title = "Open binding wizard to bind existing grammar/schema";

```

Comment:
For title I will use the same than Codelens that's to say `Bind to grammar/schema...`

Reply:
Do you think the "existing" part is more descriptive?

Reply:
yes